src/main/webapp/index.html

```diff
@@ -6,7 +6,7 @@
         <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
         <title>Kanban Board</title>
         <meta name="author" content="Gundala, Pushyami" />
```

Comment:
Since several people will maintain this as time goes by, we probably don't want the name of just one there.  We ended up stripping out the author names from the Caliper code and moving them to a "thanks" section of the README.